packages/sanity/src/core/bundles/components/panes/BundleActions.tsx

```diff
@@ -105,13 +138,13 @@ export function BundleActions(props: BundleActionsProps): ReactNode {
       data-testid={`action-add-to-${globalBundleId}`}
       text={
         isInVersion
-          ? t('bundle.action.already-in-release', {title})
+          ? t('bundle.action.discard-version', {title})
```

Comment:
COULD: pedantic but I personally feel something like this would avoid any mixups with keeping the props correct:
```
const bundleActionButtonProps = isInVersion ? {
    text: t('bundle.action.discard-version', {title}),
    icon: TrashIcon,
    tone: 'caution',
    onClick: handleDiscardVersion
} : {
    text: t('bundle.action.add-to-release', {title}),
    icon: AddIcon,
    tone: 'primary',
    onClick: handleAddVersion
}

return (
    <Button {...bundleActionButtonProps}
    />
)

```

Reply:
Good point, I'll make this change :)